### PR TITLE
Optimize description of restoring system tables

### PR DESCRIPTION
**Pull request by shichun-0415**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v6.0 (TiDB 6.0 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [x] v5.3 (TiDB 5.3 versions)
- [x] v5.2 (TiDB 5.2 versions)
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/7343
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by IANTHEREAL** [comment]
LGTM

**Review by IANTHEREAL — APPROVED** [review]

File: br/use-br-command-line-tool.md [hunk 1]
```diff
@@ -440,12 +440,12 @@ br restore full -f 'mysql.usertable' -s $external_storage_url --ratelimit 128
 
 > **警告：**
 >
-> 虽然系统表（例如 `mysql.tidb` 等）可以通过 BR 进行备份和恢复，但是部分系统表在恢复之后可能会出现非预期的状况，已知的异常如下：
+> 系统表（例如 `mysql.tidb`）可以通过 BR 备份，但系统表恢复有限制，如下系统表不能通过 BR 恢复，即便是使用了 `-filter` 设置：
```
Comment (TomShawn) [inline br/use-br-command-line-tool.md:443]:
```suggestion
> 系统表（例如 `mysql.tidb`）可以通过 BR 进行备份。但恢复系统表存在限制。即便是使用了 `-filter` 设置，也不能通过 BR 恢复以下系统表：
```

File: br/use-br-command-line-tool.md [hunk 2]
```diff
@@ -440,12 +440,12 @@ br restore full -f 'mysql.usertable' -s $external_storage_url --ratelimit 128
...
 >
-> - 统计信息表（`mysql.stat_*`）无法被恢复。
-> - 系统变量表（`mysql.tidb`，`mysql.global_variables`）无法被恢复。
-> - 用户信息表（`mysql.user`，`mysql.columns_priv`，等等）无法被恢复。
-> - GC 数据无法被恢复。
+> - 统计信息表（`mysql.stat_*`）
+> - 系统变量表（`mysql.tidb`，`mysql.global_variables`）
+> - 用户信息表（`mysql.user`，`mysql.columns_priv`，等等）
+> - [其他系统表](https://github.com/pingcap/tidb/blob/v5.3.0/br/pkg/restore/systable_restore.go#L31)
```
Comment (TomShawn) [inline br/use-br-command-line-tool.md:448]: 这里的链接 v5.3.0，以后会更新吗？

**Review by TomShawn — APPROVED** [review]

File: br/use-br-command-line-tool.md (hunk 2 above)
Comment (shichun-0415) [reply br/use-br-command-line-tool.md:448]: @IANTHEREAL PTAL

Comment (IANTHEREAL) [reply br/use-br-command-line-tool.md:448]: 理论上会

**Pull request merged**